manifests/cron.pp

```diff
@@ -8,6 +8,7 @@
   }
 
   $logrotate_path = $::logrotate::logrotate_bin
+  $logrotate_state = $::logrotate::logrotate_state
```

Comment:
`$::logrotate::logrotate_state` will always be missing as init.pp hasn't been updated.